common/force_send_fields.go

```diff
@@ -0,0 +1,63 @@
+package common
+
+import (
+	"fmt"
+	"reflect"
+	"slices"
```

Comment:
Before we upgrade to Go 1.21, use `golang.org/x/exp/slices` instead

Reply:
Thanks for the tip